pages/profile/[username].js

```diff
@@ -203,7 +202,7 @@ function Profile() {
                                         </Nav.Item>
                                         <Nav.Item>
                                             <Nav.Link eventKey="projects">
-                                                Projects
+                                                Teams
```

Comment:
Other than simply changing the tab name, the `eventKey` attribute must be changed. And similarly to the Tabs where this event key is being accessed.